cmd/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
```

Comment:
Does the indentation here mean something?

Reply:
I mean that its different from the other lines.

Reply:
Nope, just editor things.